frontend/packages/core/src/AppProvider/index.tsx

```diff
@@ -217,9 +217,12 @@ const ClutchApp = ({
                               : workflow.displayName;
 
                             const workflowLayoutProps: LayoutProps = {
-                              ...route.layoutProps,
-                              heading: route.layoutProps?.heading || heading,
                               workflow,
+                              title: heading,
+                              subtitle: route.description,
+                              variant: route.layoutProps ? route.layoutProps.variant : null,
+                              breadcrumbsOnly: route.layoutProps?.breadcrumbsOnly || false,
+                              hideHeader: route.layoutProps?.hideHeader || false,
```

Comment:
```suggestion
                              variant: route?.layoutProps?.variant || null,
                              breadcrumbsOnly: route?.layoutProps?.breadcrumbsOnly ?? false,
                              hideHeader: route?.layoutProps?.hideHeader ?? false,
```
not sure about  the `route?.layoutProps` optional chaining, can a route be missing the `layoutProps` prop?

Reply:
I agree on this change, but not with the route chaining, I think a `route` will always be defined.